e2e/testcases/apply_scoped_resource_test.go

```diff
@@ -1,138 +0,0 @@
-// Copyright 2022 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package e2e
-
-import (
-	"testing"
-	"time"
-
-	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
-	"kpt.dev/configsync/e2e/nomostest"
-	"kpt.dev/configsync/e2e/nomostest/kubevirt/v1"
-	"kpt.dev/configsync/e2e/nomostest/metrics"
-	"kpt.dev/configsync/e2e/nomostest/ntopts"
-	"kpt.dev/configsync/e2e/nomostest/retry"
-	nomostesting "kpt.dev/configsync/e2e/nomostest/testing"
-	"kpt.dev/configsync/pkg/api/configsync"
-)
-
-func TestApplyScopedResources(t *testing.T) {
-	nt := nomostest.New(t, nomostesting.Reconciliation1, ntopts.Unstructured, ntopts.SkipAutopilotCluster)
-
-	nt.Must(nt.RootRepos[configsync.RootSyncName].Copy("../../examples/kubevirt-compiled/.", "acme"))
```

Comment:
looks like this test data can be removed

Reply:
The test data is still used in https://github.com/GoogleContainerTools/kpt-config-sync/blob/be033580abc5e9f91ed4164065a87c558ab46cd6/e2e/testcases/cli_test.go#L536.

Reply:
If we refactor this test to use a different resource rather than remove it, sounds like we could also refactor that test to use the same new resource. At that point we could remove the kubevirt resources